data_write/data_write.py

```diff
@@ -853,14 +1086,14 @@ def write_tx_data():
                 parameters['int_time'] = np.float32(integration_meta.integration_time)
                 parameters['tx_pulse_len'] = np.uint32(rx_freq.pulse_len)
                 parameters['tau_spacing'] = np.uint32(rx_freq.tau_spacing)
-                parameters['num_pulses'] = np.uint32(len(rx_freq.ptab.pulse_position))
-                parameters['num_lags'] = np.uint32(len(rx_freq.ltab.lag))
                 parameters['main_antenna_count'] = np.uint32(len(rx_freq.rx_main_antennas))
                 parameters['intf_antenna_count'] = np.uint32(len(rx_freq.rx_intf_antennas))
                 parameters['freq'] = np.uint32(rx_freq.rxfreq)
-                parameters['comment'] = rx_freq.comment
+                parameters['rx_center_freq'] = integration_meta.rx_centre_freq # Sorry, we'll convert to US English here
```

Comment:
haha. I suppose we should use the most widely used English, probably US... So this is a future update then?